actions/deleteLeaflet.ts

```diff
@@ -36,5 +36,5 @@ export async function deleteDoc(permission_token: PermissionToken) {
       .where(eq(permission_tokens.id, permission_token.id));
   });
   client.end();
-  return revalidatePath("/docs");
```

Comment:
I think we can actually remove this maybe, no path needs to be validated, it's from an old way home used to work.